### PR TITLE
Refactor hearts.py to include Error Messages

### PR DESCRIPTION
**Pull request by DanielNghiem**
See https://github.com/j2kun/lonely-hearts/issues/49.

### CODE REVIEW

**Comment by j2kun** [comment]
@DanielNghiem Is this ready or WIP?

**Comment by DanielNghiem** [comment]
This PR is back to WIP.

File: hearts/game/hearts.py [hunk 1]
```diff
@@ -227,12 +239,35 @@ def set_turn_counter(self):
...
+        if len(cards) != 3:
+            error_string = message(INVALID_PASS, NOT_THREE).format(string_of_cards)
+            is_valid = False
+        elif not all(card in self.hands[player] for card in cards):
+            illegal_card = [str(card) for card in cards if card not in self.hands[player]][0]
+            error_string = message(INVALID_PASS, NOT_IN_HAND).format(string_of_cards, illegal_card)
+            is_valid = False
+        return (is_valid, error_string)
+
+    def add_to_pass_selections(self, player, cards):
```
Comment (DanielNghiem) [inline hearts/game/hearts.py]: @j2kun I will probably need to do a refactoring of the passing feature (later on) once we decide how passing works through the api.

Comment (j2kun) [inline hearts/game/hearts.py]: I agree. We will probably have to split it up into two or more steps.

**Comment by j2kun** [comment]
LGTM

**Pull request merged**